packages/renderer/src/lib/recommendation/ExtensionBanner.spec.ts

```diff
@@ -60,8 +60,24 @@ const imageBackground: IExtensionBanner = {
   },
 };
 
+const showMessageBoxMock = vi.fn();
+const telemetryTrackMock = vi.fn();
```

Comment:
```suggestion
```

just use vi.fn() inside the beforeAll
that said I think that with my changes this is now useless